index.html

```diff
@@ -1,55 +1,237 @@
-        </nav>
+      <nav class="navbar">
+        <a href="#">
+          <h1 class="nav-title">Welcome</h1>
+        </a>
+        <ul class="nav-menu">
+          <li class="nav-item"><a href="#" class="nav-link">Portfolio</a></li>
+          <li class="nav-item"><a href="#" class="nav-link">About</a></li>
+          <li class="nav-item"><a href="#" class="nav-link">Contact</a></li>
+        </ul>
+        <div class="handburger">
+          <span class="bar"></span>
+          <span class="bar"></span>
+          <span class="bar"></span>
+        </div>
+      </nav>
     </header>
 
     <section class="first-section">
-        <div class="container">
-            <h1>Hey there. I’m Daniel</h1>
-            <h1 class="desc">I’m a software developer</h1>
-            <p> I can help you build a product , feature or website Look through some of my work and experience! If you
-                like what you see and have a project you need coded,
-                don’t hestiate to contact me.</p>
-                <ul class="social">
-                    <li><a href="#"><img src="./Images//twitter.svg" alt=""></a></li>
-                    <li><a href="#"> <img src="./Images//linkedin.svg" alt=""></a></li>
-                    <li><a href="#"><img src="./Images/medium.svg" alt=""></a></li>
-                    <li><a href="#"><img src="./Images//github.svg" alt=""></a></li>
-                    <li><a href="#"><img src="./Images//angellist.svg" alt=""></a></li>
-                </ul>
-        </div>
+      <div class="container">
+        <h1>Hey there. I’m Daniel</h1>
+        <h1 class="desc">I’m a software developer</h1>
+        <p>
+          I can help you build a product , feature or website Look through some
+          of my work and experience! If you like what you see and have a project
+          you need coded, don’t hestiate to contact me.
+        </p>
+        <ul class="social">
+          <li>
+            <a href="#"><img src="./Images//twitter.svg" alt="" /></a>
+          </li>
+          <li>
+            <a href="#"> <img src="./Images//linkedin.svg" alt="" /></a>
+          </li>
+          <li>
+            <a href="#"><img src="./Images/medium.svg" alt="" /></a>
+          </li>
+          <li>
+            <a href="#"><img src="./Images//github.svg" alt="" /></a>
+          </li>
+          <li>
+            <a href="#"><img src="./Images//angellist.svg" alt="" /></a>
+          </li>
+        </ul>
+      </div>
     </section>
-    <!-- <button><img src="./Images//disabled.svg" alt=""></button>
+    <button><img src="./Images//disabled.svg" alt="" /></button>
     <section class="second-section">
+      <div class="container">
+        <h1 class="recent-header">My Recent Works</h1>
+        <hr />
+        <div class="boxes">
+          <div class="box">
+            <div></div>
+            <div class="box-menu">
+              <h1 class="title">Multi-Post Stories Gain+Glory</h1>
+              <ul class="box-list">
```

Comment:
- [Optional] Dear @danutibe07, Good job applying the `hx` tag here. but for semantic reasons please use only one `h1 tag` on a single page. everything else should be in `h2-h6`, This gives screen readers the advantage of knowing what is the primary content of the page is.